tests/gnu/testlet/MIDletTestlet.java

```diff
@@ -0,0 +1,4 @@
+package gnu.testlet;
+
+public interface MIDletTestlet extends Testlet {
```

Comment:
Is there something you plan to put in *MIDletTestlet*? It isn't clear why this is necessary when it doesn't actually change or add to the *Testlet* class.